apis/python/tests/test_from_xenium.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+import pytest
+
+import tiledbsoma as soma
+
+from ._util import ROOT_DATA_DIR
+
+spatial_io = pytest.importorskip("tiledbsoma.io.spatial")
+
+
+@pytest.fixture(scope="module")
+def xenium_v1_path():
+    """Fixture that checks the example Xenium v1 dataset exists."""
+    xenium_path = ROOT_DATA_DIR / "example-xenium-v1"
+    if not os.path.isdir(xenium_path):
+        raise RuntimeError(
+            "Missing 'data/example-xenium-v1' directory. Try running `make data` "
+            "from the TileDB-SOMA project root directory."
+        )
+    for filename in [
+        "cell_feature_matrix.h5",
+        "cells.parquet",
+        "cell_boundaries.parquet",
+        "nucleus_boundaries.parquet",
+        "experiment.xenium",
+    ]:
+        if not os.path.isfile(xenium_path / filename):
+            raise RuntimeError(
+                f"Missing file 'data/example-xenium-v1/{filename}'. Try removing "
+                f"the directory 'data/example-xenium-v1' and re-running `make data'"
+                f"from the project root directory."
+            )
+
+    return xenium_path
+
+
+@pytest.mark.skip(reason="Missing test dataset")
```

Comment:
We need to add the test data to the test data repo before we fully release the Xenium reader. I added [a shortcut story](https://app.shortcut.com/tiledb-inc/story/65476/add-xenium-test-data) for this.